src/features/profile/berths/queries.ts

```diff
@@ -0,0 +1,49 @@
+import { gql } from 'graphql-tag';
+
+const BERTH_APPLICATION_NODE = gql`
+  fragment berthApplicationNodeFragment on BerthApplicationNode {
+    id
+    createdAt
+    harborChoices {
+      harbor {
+        id
+        properties {
+          name
+          availabilityLevel {
+            id
+            title
+          }
+          electricity
+          gate
+          lighting
+          wasteCollection
+          water
+        }
+      }
+    }
+  }
+`;
+
+export const BERTHS_QUERY = gql`
+  query BERTHS {
+    myProfile {
+      berthApplications {
+        edges {
+          node {
+            ...berthApplicationNodeFragment
+          }
+        }
+      }
+    }
+  }
+
+  ${BERTH_APPLICATION_NODE}
+`;
```

Comment:
We already have a profile query, should this fragment be appended to that query?

Reply:
That makes sense, let's keep it this way 👍